components/Link/hooks.ts

```diff
@@ -31,7 +32,21 @@ export const useNormalizedHref = (href: string) => {
     ? href.substring(basePath.length)
     : href;
 
-  const { scope } = useContext(DocsContext);
+  const scopes = {
+    oss: true,
+    cloud: true,
+    enterprise: true,
+  };
+
+  let scope: ScopeType = useContext(DocsContext).scope;
+
+  const { query } = splitPath(href);
+
+  // If a valid scope is provided via query parameter, adjust the
+  // link to navigate to that scope.
+  if (query.hasOwnProperty("scope") && scopes[query.scope] == true) {
```

Comment:
You can actually replace this code with `scopeValues` too(https://github.com/gravitational/docs/blob/main/layouts/DocsPage/types.ts#L6).

Something like `scopeValues.includes(scope)`.

Reply:
Thanks! Made this change in  https://github.com/gravitational/docs/pull/51/commits/092e6991e166545974d1b6d9423b3795d436a249